contracts/sol6/KyberNetwork.sol

```diff
@@ -758,11 +758,13 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
             hint
         );
 
+        require(tradeData.tradeWei <= MAX_QTY, "tradeWei > MAX_QTY");
```

Comment:
Ethereum's total supply is 111 million = `1e26`...

IMO this check is somewhat unnecessary, because reserves won't hold that much ETH to transfer to the network, so the trade will revert anyway.

Reply:
Yes

but it helps later when calculating reverse trade.
can have clear assumption on quantity